apps/web/src/styles/useColor.test.tsx

```diff
@@ -0,0 +1,68 @@
+import { useColorMode } from "@chakra-ui/react";
+import { renderHook } from "@testing-library/react-hooks";
+import { type PropsWithChildren } from "react";
+
+import { dark, light } from "./colors";
+import { useColor } from "./useColor";
+import { LogoDarkIcon, LogoLightIcon } from "../assets/icons";
+import { UmamiTheme } from "../providers/UmamiTheme";
+
+jest.mock("@chakra-ui/react", () => ({
+  ...jest.requireActual("@chakra-ui/react"),
+  useColorMode: jest.fn(),
+}));
+
+jest.mock("../assets/icons", () => ({
+  LogoDarkIcon: () => "logo-dark-icon",
+  LogoLightIcon: () => "logo-light-icon",
+}));
+
+describe("useColor", () => {
+  const mockUseColorMode = useColorMode as jest.Mock;
```

Comment:
it's better to use jest.mocked(useColorMode) as it is type-safe

Reply:
this is indeed true, but in our case we don't need full type safety, as we will only mock part of the `useColorMode` output

Reply:
okay.png